elements/lisk-p2p/src/peer.ts

```diff
@@ -88,9 +89,9 @@ type SCServerSocketUpdated = {
 } & SCServerSocket;
 
 export enum ConnectionState {
-	CONNECTING = 0,
-	CONNECTED = 1,
-	DISCONNECTED = 2,
+	CONNECTING = 'connecting',
```

Comment:
why change this to string now?

Reply:
To make debugging simpler. Also it's not part of the protocol; it's only used internally by the P2P library so it's non-breaking.